src/txmempool.cpp

```diff
@@ -652,6 +652,7 @@ void CTxMemPool::check(const CCoinsViewCache *pcoins) const
 
     checkNullifiers(SPROUT);
     checkNullifiers(SAPLING);
+    checkNullifiers(ORCHARD);
```

Comment:
This isn't the only consistency check missing in this function. You'll see above that we need to (as we do with orchard and sapling) check that Orchard nullifiers _don't_ exist in `pcoins` (which is the backing `CCoinsViewCache`) and that the anchors for Orchard bundles _do_ exist in `pcoins`.

Reply:
This is now addressed in #5804.